src/NSwag.Commands/NSwagDocumentBase.cs

```diff
@@ -515,12 +525,6 @@ private static string TransformLegacyDocument(string data, out bool saveFile)
                 saveFile = true;
             }
 
-            if (data.Contains("\"noBuild\":") && !data.ToLowerInvariant().Contains("UseDocumentProvider".ToLowerInvariant()))

```

Comment:
Couldn't find a property `UseDocumentProvider` anywhere so this seemed unnecessary

Reply:
I think this is gone, correct